src/types/GetTrieHashesReq.ts

```diff
@@ -1,11 +1,13 @@
 import { VectorBufferStream } from '../utils/serialization/VectorBufferStream'
 import { TypeIdentifierEnum } from './enum/TypeIdentifierEnum'
+import { verifyPayload } from './ajv/Helpers'
+import { AJVSchemaEnum } from './enum/AJVSchemaEnum'
 
 export type GetTrieHashesRequest = {
   radixList: string[]
 }
 
-const cGetTrieHashesReqVersion = 1
+export const cGetTrieHashesReqVersion = 1
```

Comment:
again, this should not be exported since it is not being used outside the file.

Reply:
Fixed